src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './styles.scss';
+import {Nav, NavItem, NavLink} from "shards-react";
+import {FaBars, FaHome, FaMoneyBillWave} from 'react-icons/fa';
+// FaBars FaHome FaMoneyBillWave
```

Comment:
Remove pls

Reply:
@alphakennyn but if I remove it then wouldn't it compile incorrectly as the icons depend on it? What should I use instead then? It's place holders should I remove them when we have the assets or do we already have them?

Reply:
I was referring to `line 5` which is commented-out code. Keep our code clean